docker/jupyter/Dockerfile

```diff
@@ -16,15 +18,12 @@ COPY docker/jupyter/js /jupyter/js
 
 RUN apt-get -qq update
 
-COPY /docker/ci/install-python.bash /jupyter
-RUN /jupyter/install-python.bash
-
-COPY docker/ci/install-pips.bash /jupyter/
-RUN /jupyter/install-pips.bash
```

Comment:
If we don't install pips, we can't use `numpy` and other pips in `Jupyter Notebook`, are we intend to do so?

Reply:
Maybe users can install the Python tools using `pip install` in Jupyter Notebook easily.